docker/docker-compose.yml

```diff
@@ -75,17 +75,13 @@ services:
       - maildev
     image: automattic/jetpack-wordpress-dev:latest
     volumes:
-      - ..:/var/www/html/wp-content/plugins/jetpack
-      ## Kludge for not having docker contain recursive stuff
-      ## You will see on your filesystem that this dir gets created
+      ## WordPress and its plugins and themes should be defined in docker/compose-volumes.yml{.sample}, not here!
+      ## That avoids haing to define them both here and in the "sftp" service.
```

Comment:
```suggestion
      ## That avoids having to define them both here and in the "sftp" service.
```

Reply:
Thanks! I'll squash this into the update I'm about to push for the other comments.